src/vue/backend/src/App.vue

```diff
@@ -1,12 +1,131 @@
 <template>
-  <idx-block className="app">
-    <idx-block className="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
+    <idx-block :className="{app: isActive, 'nav-is-collapsed': navCollapsed}">
+        <Header>
+            <template v-slot:toggle>
+                <idx-button
+                    @click="toggleCollapse()"
+                    customClass="toggle-nav"
+                    :aria-expanded="navCollapsed ? 'false' : 'true'">
+                    <img src="@/assets/plus.svg" height="30" alt="IDX Broker" loading="lazy">
```

Comment:
'IDX Broker' as alt text doesn't make sense here. No alt tag is necessary since it doesn't add information to the screen reader since it's an icon alongside text.